package.json

```diff
@@ -3,21 +3,17 @@
   "version": "0.1.1",
   "description": "Command Line Interface (CLI) for Open Translators to Things.",
   "bin": {
-    "opent2t": "./index.js"
+    "opent2t-cli": "./index.js",
+    "opent2t-ui": "./reference-app/bin/ref-app"
   },
   "dependencies": {
-    "angular": "^1.6.1",
-    "angular-resizable": "^1.2.0",
-    "bootstrap": "^3.3.7",
     "colors": "^1.1.2",
     "commander": "^2.9.0",
+    "electron": "^1.6.2",
     "express": "^4.14.0",
-    "font-awesome": "^4.7.0",
     "glob": "^7.1.1",
     "inquirer": "^1.1.3",
-    "jquery": "^3.1.1",
     "open": "0.0.5",
-    "opent2t": "^1.0.1",
```

Comment:
No, it uses the opent2t module from the node_modules in the current directory.

---
In reply to: [105346247](https://github.com/openT2T/opent2t-cli/pull/28#discussion_r105346247) [](ancestors = 105346247)